api/v1alpha3/machinehealthcheck_webhook.go

```diff
@@ -46,6 +47,11 @@ func (m *MachineHealthCheck) Default() {
 		defaultMaxUnhealthy := intstr.FromString("100%")
 		m.Spec.MaxUnhealthy = &defaultMaxUnhealthy
 	}
+
+	if m.Spec.NodeStartupTimeout == nil {
+		defaultNodeStartupTimeout := metav1.Duration{Duration: 10 * time.Minute}
```

Comment:
Should this be turned into a constant?

Reply:
I don't think I can turn this into a `const` because it's a struct literal, I could turn the `10 * time.Minute` into a `const` though, WDYT?
```
const initializer "k8s.io/apimachinery/pkg/apis/meta/v1".Duration literal is not a constant
```

Reply:
ah, good call, that works for me :)